src/applications/disability-benefits/all-claims/utils/submit.js

```diff
@@ -356,11 +356,17 @@ export const addForm4142 = formData => {
     }),
   };
   delete clonedData.limitedConsent;
-  delete clonedData.providerFacility;
+  if (!clonedData.syncModern0781Flow) {
+    delete clonedData.providerFacility;
+  }
   return clonedData;
 };
 
 export const addForm0781 = formData => {
+  if (formData.syncModern0781Flow) {
+    return formData;
+  }
+
```

Comment:
Just curious, what is this function doing? 

the name of the function is `addForm0781` but it returns right away if there is `syncModern0781Flow` true/present.

Reply:
Oops. Should return early if `formData.syncModern0781Flow === false`. Will update. Thanks.

Reply:
I dont think that is exactly what I was asking. I think either `formData.syncModern0781Flow === true` or `formData.syncModern0781Flow` will function the same/similar here. I am more asking.... why is this function called `addForm0781` when, in the times I would expect to need to add form data (`formData.syncModern0781Flow` is true-ish) we are actually just returning right away.

I am just confused what this function is or does

Reply:
Oh. The function prepares all data pertaining to the current 0781 form. To avoid including any unnecessary 'form0781' submission data, the `formData` is returned early, and the submit-transformer continues on to add new 0781 data, if any.

Reply:
Correct. Good question as I didn't explicitly provide justification.